src/template.rs

```diff
@@ -377,12 +377,18 @@ impl Template {
                     {
                         let (line_no, col_no) = parser.input().line_col(prev_end);
                         if token.rule == Rule::raw_block_end {
-                            let text = &source[prev_end..token.start];
+                            let mut text = &source[prev_end..token.start];
+                            if text.starts_with("\\{{") {
```

Comment:
hi @softprops , what if the `\\{{` is not at beginning of a raw_text, like `hello \\{{world}}`